### DISCUSSION TITLE
Suggestion for hotkeys

### DISCUSSION THREAD

**Discussion by ColeWayneJones** [SnosMe/awakened-poe-trade#1256] [Ideas]
Was wondering if it would be possible for each "Stash Search" you create let's call the first one A and the second one B

If you have a hotkey for Stash Search A for a few regex
"attack speed" hotkey 1
"cast speed" hotkey 2

And then for Stash Search B you have another few hotkeys for regex
"critical strike" hotkey 1
"damage over time" hotkey 2

Is it possible to add an option that you can click on/off in hotkeys section where the stash search that is active is the one that has it's hotkeys applied.  So you can have multiple Stash Searches that all share hotkeys but only one that is active at a given time?

As it is currently if you want to use the same hotkeys across multiple of them you have to unbind current hotkeys in one and then redo them in the next.

**Comment by SnosMe** [comment]
interesting idea

**Reply by ColeWayneJones** [reply]
How hard would it be to implement?
